beetmoverscript/templates/firefox_nightly.yml

```diff
@@ -129,6 +129,11 @@ mapping:
       destinations:
         - {{ upload_date }}-{{ branch }}/mozharness.zip
         - latest-{{ branch }}/mozharness.zip
+    buildhub.json:
+      s3_key: {{ upload_date }}-{{ branch }}/firefox-{{ version }}.{{ locale }}.{{ platform }}.buildhub.json
```

Comment:
`s3_key` is solely used in checksums generation. Whatever we end up placing here, will be landing in the SHA512SUMS files (e.g. [here](https://archive.mozilla.org/pub/firefox/candidates/61.0b14-candidates/build1/SHA256SUMS)). So in this case, it's better if we simply keep the `firefox-{{ version }}.{{ locale }}.{{ platform }}.buildhub.json` part without its leading `upload_date ...` prefix.

Reply:
This is still wrong. It should be `firefox-{{ version }}.{{ locale }}.{{ platform }}.buildhub.json` instead